### PR TITLE
EZP-29864: Spelling mistake in Content Type alert warnings

### PR DESCRIPTION
**Pull request by pawbuj**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [EZP-29864](https://jira.ez.no/browse/EZP-29864)
| **Bug/Improvement**| yes
| **New feature**    | no
| **Target version** | `6.7`
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | no

**TODO**:
- [x] Implement feature / fix a bug.
- [x] Implement tests.
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,4 @@ composer.phar
 var/
 composer.lock
 .php_cs.cache
+.idea
```
Comment (alongosz) [inline .gitignore]: @pawbuj we don't exclude developer environment-specific files in .gitignore. Please use global gitignore to exclude PhpStorm directory. I keep mine in `.config/git/ignore`, but that might be OS-dependent. Here's [GitHub recommendation](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) how to do it.

Comment (pawbuj) [reply .gitignore]: ok

**Comment by lserwatka** [comment]
Congratulations @pawbuj Your first kernel PR 🥇

**Comment by lserwatka** [comment]
You can merge it up to 6.13, 7.3 and master branches.

**Pull request merged**